game/view/sector.lua

```diff
@@ -63,10 +66,25 @@ function SectorView:init(route)
 
 end
 
+function SectorView:hasPendingVFX()
+  return not Util.tableEmpty(self.vfx.offset)
+end
+
 function SectorView:lookAt(target)
   self.target = target
 end
 
+function SectorView:addVFX(extra)
+  if extra.type == 'body_moved' then
+    local body, i, j = extra.body, unpack(extra.origin)
+    local i0, j0 = body:getPos()
+    local offset = {i - i0, j - j0}
+    self.vfx.offset[body] = offset
+    self:addTimer(nil, MAIN_TIMER, "tween", 0.10, offset, {0, 0}, "in-out-quad",
```

Comment:
This is kinda slow, esp when each body moves at a time. I tested with 0.05 and it looked fine. The faster the better. The flickering of the actor hud is bothersome but we can hide it by default so it doesn't keep flickering all the time.